src/index.js

```diff
@@ -36,6 +40,7 @@ import 'styles/main.scss';
       const dataAttrOptions = {
         selectionText: $selection.data('selection-text'),
         textPlural: $selection.data('text-plural'),
+        placeHolderText: $this.data("placeholder"),
```

Comment:
strings must use single quotes. you can check for lint errors running `yarn lint`